grails-app/assets/javascripts/coreReportService.js

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
It's probably best to put this into the existing ReportService.js.
The term "Core" is no longer used in the new program so we don't really need a new service for it.